### PR TITLE
fix pedal import/export for MusicXML

### PR DESCRIPTION
**Pull request by rettinghaus**
This PR contains only the fixes to MusicXML import/export from #20219.

### CODE REVIEW

**Review by cbjeukendrup — APPROVED** [review]

File: src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp [hunk 1]
```diff
@@ -3552,6 +3552,8 @@ void MusicXMLParserDirection::pedal(const QString& type, const int /* number */,
             p->setBeginText(u"<sym>keyboardPedalPed</sym>");
             p->setContinueText(u"(<sym>keyboardPedalPed</sym>)");
         } else {
+            p->setBeginText(u"");
+            p->setContinueText(u"");
```
Comment (Jojo-Schmitz) [inline src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp:3556]: This is different from #20219

Comment (Jojo-Schmitz) [reply src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp:3556]: Sorry, just ascreenshot from my mobile

Comment (rettinghaus) [reply src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp:3556]: Exactly, this is just the fixing part that should go into 4.2.

**Review by oktophonie — APPROVED** [review]

**Comment by Jojo-Schmitz** [comment]
Seems something is wrong with this PR (and its counterpart for master too):
Take the attached score, export to MusicXML (attachetd too), import in Mu3 (the official way from Mu4 back to Mu3), gives:

Fatal error: line 108 column 73 Content of attribute type does not match its type definition: String content is not listed in the enumeration facet..

```XML
      <direction placement="below">
        <direction-type>
          <pedal type="resume" line="yes" sign="yes" default-y="-65.00"/>
          </direction-type>
        </direction>
 ```
It is the `type="resume"` it complains about 
[pedal.zip](https://github.com/musescore/MuseScore/files/13628820/pedal.zip)

**Comment by Jojo-Schmitz** [comment]
Might not be reletae to this PR, as the issue exists with 4.2.0 Beta already, which IMHO is older than this PR's merg into the 4.2.0 branch?
Anyway, I've opened issue #20395 for this

**Pull request merged**